src/plugins/intel_cpu/src/transformations/tpp/x64/pass/fuse_tpp_to_equations.cpp

```diff
@@ -82,7 +82,15 @@ bool FuseTPPToEquations::fuse_from_root(const NodePtr& root, const std::shared_p
         kv.second = equation;
     replace_nodes(m, {}, node_replace_map);
     for (const auto& in : equation->inputs()) {
-        ov::snippets::lowered::PortDescriptorUtils::set_port_descriptor(in, root_subtensor);
+        auto subtensor = root_subtensor;
+        if (in.get_partial_shape().size() < 2) {
```

Comment:
The number `2` is magic number here 😄 

Can we create subtensor using ranks of `root_subtensor` and `in.get_partial_shape()` (smth like `std::min(root_subtensor.size(), in.get_partial_shape().size())`) ?
I mean if `root_subtensor.size() = 3` and `in.get_partial_shape().size() = 2`, we won't handle correctly subtensor for this input.

Reply:
Agree, added comparison for `root_subtensor.size()` and `in.get_partial_shape().size()`